swagger_templates/python/api.mustache

```diff
@@ -126,66 +107,52 @@ class {{classname}}(object):
 {{/vendorExtensions.x-isi-url-encode-path-param}}
```

Comment:
This is the portion of code that I added for tagging a parameter with "__x-isi-url-encode-path-param__", the rest of these changes are due to reverting from the latest upstream version of the template to the template that we were using prior to my last commit.